src/styles/utils/placeholders.scss

```diff
@@ -0,0 +1,15 @@
+%day__style {
+  box-sizing: border-box;
+  border-radius: 10px;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+
+  background-color: #eee;
+  border: 1px solid black;
+
+  width: $width;
+  height: $height;
+
+
+}
```

Comment:
there should not be empty lines before } and should be one empty line after }.

Reply:
Fixed, thx😊.